### ISSUE TITLE
Error Trying to stall using Composer Magento 2.1.7

### ISSUE THREAD

**Issue by brendanbell** [fontis/australia-mage2#8]
Hi Chris,

Have installed a clean copy of magento 2.1.7

Trying to install by composer, but its looking for the customergroup. I had a look and you have this, but reading the notes for that extension it says its not a magento 2 extension

![image](https://user-images.githubusercontent.com/1163263/28742039-69bb23b0-7468-11e7-807f-55133479a3be.png)

thanks
Brendan

**Comment by chnorton** [comment]
Would you be able to post a copy of your `composer.json` file? It seems that there's something going wrong when it's working out which versions to install, so it would help to see the specific versions that are being requested.

**Comment by brendanbell** [comment]
sure

[composer.txt](https://github.com/fontis/australia-mage2/files/1186060/composer.txt)

**Comment by chnorton** [comment]
I wasn't able to reproduce this myself when I run `composer require fontis/australia-mage2` using your `composer.json` file. To work around it, could you try manually adding this to your `composer.json`:

`"fontis/australia-mage2": "^1.0"`

Then run:

`composer update fontis/australia-mage2 fontis/customergroupaccess-mage2`

And see if that helps?

**Comment by brendanbell** [comment]
Cool,
Running this produced the following:

![fontis-error](https://user-images.githubusercontent.com/1163263/28761466-189a8b66-75f2-11e7-9310-72684e93cc74.png)

Should I continue?

**Comment by brendanbell** [comment]
So I kicked on and the following error occured

![fontis-error-2](https://user-images.githubusercontent.com/1163263/28761608-22fcc1c2-75f3-11e7-89b3-59aab51829ef.png)

**Comment by chnorton** [comment]
Right, so that would likely be because it's trying to get it from repo.magento.com and you haven't added the extension to your account on marketplace. You can either do that, [add a Composer repository for the GitHub repository](https://getcomposer.org/doc/05-repositories.md#vcs), or [get it off our website](https://www.fontis.com.au/australia-magento-extension) and [set it up that way](https://www.fontis.com.au/composer-faq).
